apps/src/templates/teacherHomepage/GradientNavCard.jsx

```diff
@@ -55,29 +55,35 @@ const styles = {
 
 const GradientNavCard = React.createClass({
   propTypes: {
-    cardData: React.PropTypes.shape({
-      title: React.PropTypes.string.isRequired,
-      description: React.PropTypes.string.isRequired,
-      image: React.PropTypes.string.isRequired,
-      buttonText: React.PropTypes.string.isRequired,
-      link: React.PropTypes.string.isRequired
-    })
+    title: React.PropTypes.string.isRequired,
+    description: React.PropTypes.string.isRequired,
+    image: React.PropTypes.string.isRequired,
+    buttonText: React.PropTypes.string.isRequired,
+    link: React.PropTypes.string.isRequired
   },
 
   render() {
-    const { cardData } = this.props;
+    const { title, description, buttonText, link } = this.props;
 
     return (
       <div style={styles.card}>
         <img src={require('../../../static/navcard-placeholder.png')} style={styles.image}/>
```

Comment:
Realize this is from an older commit, but I'd prefer to see any requires/imports at the top of the file.